common/proto-options/inheritance/decrypt_data_with_pin.options

```diff
@@ -1,3 +1,3 @@
-inheritance.DecryptDataWithPinMessagesResponse.plain_data type:FT_STATIC max_count:5  fixed_length:false
-inheritance.DecryptDataWithPinInitiateRequest.encrypted_data type:FT_STATIC max_size:6000 fixed_length:false
+inheritance.DecryptDataWithPinMessagesResponse.plain_data type:FT_STATIC max_count:1  fixed_length:false
+inheritance.DecryptDataWithPinInitiateRequest.encrypted_data type:FT_STATIC max_size:60 fixed_length:false
```

Comment:
I think these were just temporary changes.

Reply:
yes have fixed them in decryption chunking branch, merging should remove them